src/authentication/middleware/store_access_token_in_auth_service.js

```diff
@@ -5,8 +5,9 @@ export default (req, res, next) => {
   const authorizationHeaderValue = getRequestHeader('Authorization')
 
   // TODO: extract the token from the Authorization header value
-  const token = ''
-
-  setAccessToken(token)
-  next()
+  if (authorizationHeaderValue) {
+    const token = authorizationHeaderValue.split('')[1]
+    setAccessToken(token)
+    next()
+  }
```

Comment:
You should always call `next`, even if there is no Authorization header. Move this call outside of the `if` statement.
```suggestion
  }
  
  next()
```